api/proto/canary/gcs.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto3";
+package proto.canary;
+
+option go_package = "github.com/spinnaker/kleat/api/client/canary";
+
+// Configuration for the GCS canary integration. If enabled, you must
+// also configure at least one canary.GoogleAccount with a list of
+// supportedTypes that includes canary.SupportedType.CONFIGURATION_STORE and/or
+// canary.SupportedType.OBJECT_STORE.
```

Comment:
This is another confusing thing worth calling out: Halyard doesn't actually expose commands to edit any accounts' `SupportedType`s. It defaults it for metrics-only integrations, which I handled by defaulting those types in Kayenta itself. When you enable GCS, S3, or Stackdriver using Halyard, it automatically adds the appropriate types to each configured account's list. The reason I exposed the field on `google` and `aws` is because I think there is a reasonable use case for using different accounts for `CONFIGURATION_STORE` vs. `OBJECT_STORE`. This won't impact existing Halyard users since they will already have these specified in their halconfig, but I've added explicit documentation for GCS, S3, and Stackdriver so this is clear for new users who will need to fill in supported types manually.

Reply:
I think this makes sense! While it may take a few minutes for a user to understand this the first time they set it up, I think it's reasonable to expect that level of understanding and will probably be helpful to those users in the long run.